README.md

```diff
@@ -90,6 +90,7 @@ kind: QuayEcosystem
 metadata:
   name: example-quayecosystem
 spec:
+  imagePullSecretName: redhat-pull-secret
```

Comment:
@itewk can you remove this reference to the `imagePullSecretName`?

Reply:
done.